docs.cask.co/www/404.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+
+<html xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta content="Cask Data, Inc." name="author" />
+    <meta content="Missing Page" name="description" />
+    <meta content="Copyright © 2015-2016 Cask Data, Inc." name="copyright" />
+    <meta name="robots" content="noindex" />
+
+    <!-- Google Tag Manager start -->
+    <script>dataLayer = [];</script>
+    <noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-KWLFGH"
+    height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','GTM-KWLFGH');</script>
+    <!-- Google Tag Manager end -->
+
+    <title>Page Not Found (404 error message) &mdash; Cask Data Documentation</title>
+    
+    <link rel="stylesheet" href="cdap/current/en/_static/cdap.css" type="text/css" />
+    <link rel="stylesheet" href="cdap/current/en/_static/pygments.css" type="text/css" />
+
+    <script type="text/javascript">
+      var DOCUMENTATION_OPTIONS = {
+        URL_ROOT:    './',
+        VERSION:     '',
+        COLLAPSE_INDEX: false,
+        FILE_SUFFIX: '.html',
+        HAS_SOURCE:  false
+      };
+    </script>
+    <script type="text/javascript" src="cdap/current/en/_static/jquery.js"></script>
+    <script type="text/javascript" src="cdap/current/en/_static/underscore.js"></script>
+    <script type="text/javascript" src="cdap/current/en/_static/doctools.js"></script>
```

Comment:
Should any of this be at the root level? Maybe under `resources/scripts`? Seems like yes, to me.